src/marqo/README.md

```diff
@@ -35,14 +35,35 @@ docker run --detach --name vespa --hostname vespa-tutorial \
 ```bash
 (cd scripts/vespa_local && zip -r - * | curl --header "Content-Type:application/zip" --data-binary @- http://localhost:19071/application/v2/tenant/default/prepareandactivate)
 ```
-
 You can verify that Vespa has been set up correctly by visiting `http://localhost:8080` in your browser.
 
+5. We need to install Java Development Kit and Maven to build the jar file for the custom searchers.
+
+5a. Install Java Development Kit: 
+You can install the Java Development Kit (JDK) by following the instructions [here](https://docs.oracle.com/en/java/javase/22/install/overview-jdk-installation.html). 
+Post installation, remember to set JAVA_HOME Environment Variable and to add Java to the PATH Environment Variable.
+You can verify the installation by running the following command:
+```bash
+java -version
+```
+5b. Install Maven 
+You can install Maven by following the instructions [here](https://maven.apache.org/install.html).
+Similar to setting Java, you need to add Maven to the PATH Environment Variable.
+Verify Maven installation by running the following command: 
+```bash
+mvn -version
+```
+Post this you need to create a jar file, cd into the vespa directory in your local marqo repository, and run 
```

Comment:
The wording is confusing. How do I create a jar file?

Reply:
changed